Stripe/STPAPIClient.m

```diff
@@ -813,4 +815,16 @@ - (void)createPaymentMethodWithParams:(STPPaymentMethodParams *)paymentMethodPar
 
 }
 
+#pragma mark - FPX
+
+- (void)retrieveFPXBankStatusWithCompletion:(STPFPXBankStatusCompletionBlock)completion {
+    [STPAPIRequest<STPFPXBankStatusResponse *> getWithAPIClient:self
```

Comment:
ooc, does this handle errors/exceptions?

Reply:
No, I don't think there's anything useful we could do with such an error. We could log them to the console, I guess.

Reply:
it does in the sense that we will generate an error if the response is not parseable for instance

Reply:
I can't determine if we have any standardized way to communicate these sort of errors up to users. `retrieveFPXBankStatusWithCompletion` will return an error to the completion block if the response isn't parseable, but the BankSelectionViewController can't do anything sensible with that other than silently failing. We could `NSLog()` a warning, but we don't have any precedent for that.